### PR TITLE
Adds code of conduct

### PR DESCRIPTION
**Pull request by simonswine**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

### CODE REVIEW

**Comment by munnerz** [comment]
You should include the release note block with `NONE` in it, or alternatively add the /release-note-none label to get around the release not problems

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an
+appointed representative at an online or offline event. Representation of a
+project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at hello@jetstack.io or
+christian@jetstack.io. The project team will review and investigate all
```
Comment (munnerz) [inline CODE_OF_CONDUCT.md]: Should we be consistent with the point of contact between projects? i.e. a designated CoC team?

Comment (simonswine) [reply CODE_OF_CONDUCT.md]: I guess maybe an alias to helllo@jetstack.io could be perfect. Not too sure, what is @mattbates opinion?

**Comment by simonswine** [comment]
@munnerz please lgtm again, I have added an alias for it for jetstack.io

**Pull request merged**